api/v1alpha1/condition_consts.go

```diff
@@ -161,6 +161,10 @@ const (
 	// and the published item is uploading to the target location.
 	UploadingReason = "Uploading"
 
+	// UploadItemIDInvalidReason documents that the VM publish task result
+	// returns an invalid Item id.
+	UploadItemIDInvalidReason = "ItemIDInvalid"
```

Comment:
@yi0909 Please relocate all condition constants related to a specific resource type to that resource type's file.

Reply:
@akutz, is this the recommended practice for having the condition consts with the respective types?  IIRC, we followed the CAPI model of putting all conditions in one file.